gensrc/script/doris_builtins_functions.py

```diff
@@ -2121,6 +2121,9 @@
         [['to_ipv6_or_default'], 'IPV6', ['STRING'], 'ALWAYS_NOT_NULLABLE'],
         [['to_ipv6_or_null'], 'IPV6', ['VARCHAR'], 'ALWAYS_NULLABLE'],
         [['to_ipv6_or_null'], 'IPV6', ['STRING'], 'ALWAYS_NULLABLE'],
+        [['ipv4_to_ipv6'], 'IPV6', ['IPV4'], 'DEPEND_ON_ARGUMENT'],
+        [['cut_ipv6'], 'VARCHAR', ['IPV6'], 'DEPEND_ON_ARGUMENT'],
+        [['cut_ipv6'], 'STRING', ['IPV6'], 'DEPEND_ON_ARGUMENT'],
```

Comment:
this function signature is not same with nereids planner? should have three args?

Reply:
yes, it's a mistakes, thanks for code review